docs/dev/hybrid_script.rst

```diff
@@ -0,0 +1,76 @@
+Hybrid Frontend Developer Guide
+===============================
+
+If you are a developer:
+
+1. who is trying writing some preliminary patterns that have not been supported by TVM yet,
+maybe `language ref <../langref/hybrid_script.rst>`_ is a better place for you.
```

Comment:
Use ref. see http://www.sphinx-doc.org/en/stable/markup/inline.html cross reference arbitrary locations

Reply:
This is a ref to a file instead of an anchor. Can I still use label to do it? I suppose label is only for intra file instead of inter.

Reply:
label works for inter file